docs/SQS.md

```diff
@@ -108,7 +108,7 @@ The SQSReceiver class can be directly instantiated and has the following paramet
   - Whether to terminate the message visibility timeout when SQSReceiverMessage.RollbackMessageAsync is called. Terminating the message visibility timeout allows the message to immediately become available for queue consumers to process.
 
 ```c#
-IReceiver receiver = new SQSReceiver("MyReceiver", "https://sqs.us-west-2.amazonaws.com/123456789012/your_queue_name",
+IReceiver receiver = new SQSReceiver("MyReceiver", new Uri("https://sqs.us-west-2.amazonaws.com/123456789012/your_queue_name"),
```

Comment:
👍  Could you also put the Uri changes in the CHANGELOG when you create it for the release?

Reply:
Er you could just put them in now.  Forgot it was in the initial prs.  And if I read it I'll see you already did so.:)

Reply:
It's actually already in the CHANGELOG in the unreleased section right now for these updates.

Reply:
And now I noticed that you already noticed it while I was double checking any typing up my original response lol

Reply:
ha